google-youtube-upload.html

```diff
@@ -393,6 +422,30 @@
         value: Boolean('MediaRecorder' in window &&
                        'mediaDevices' in navigator &&
                        'getUserMedia' in navigator.mediaDevices)
+      },
+
+      /**
+       * The [MediaStream audio constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/audio).
+       */
+      audio: {
+        type: Object,
```

Comment:
I'm on the fence on what the type should be here. `mediaDevices.getUserMedia` takes `boolean| MediaTrackConstraints`. @cdata suggestions?